.github/workflows/qaqc.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       with:
         check_filenames: true
         check_hidden: false
-        skip: '*.js,qaqc.yml,acp_lidar_box.geojson'
+        skip: '*.js,qaqc.yml,acp_lidar_box.geojson,.codespellignore'
```

Comment:
@scottyhq - I am suspicious that the QA/QC workflow didn't run after this change. Let's keep an eye on this